src/rust/engine/options/src/fromfile.rs

```diff
@@ -51,11 +51,16 @@ fn try_deserialize<'a, DE: Deserialize<'a>>(
     }
 }
 
-pub struct FromfileExpander {}
+#[derive(Clone, Debug)]
+pub struct FromfileExpander {
+    build_root_str: String,
```

Comment:
Given this seems to just be used with a `Path`, did you consider using a `PathBuf` here?

I know that there's other uses of `build_root_string` in `OptionParser::new`, but seems like we could at last have this one be "cleaner" for paths?

Reply:
Yeah, not sure why I did it that way, probably it got entangled with that string use in the code or in my mind. Will switch to paths. Good catch.

Reply:
Done, great suggestion.